types/dos2unix/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "@types/minimatch": "*"
+        "@types/glob": "~5.0.0"
```

Comment:
Not `^` for these?

Reply:
Seems not

https://github.com/JamesMGreene/node-dos2unix/blob/master/package.json#L33

`"@types/glob": "~5.0.0"` because that's the earliest version in the `@types` repo